Packs/Core/Playbooks/playbook-Identity_Analytics_-_Alert_Handling.yml

```diff
@@ -1644,20 +1640,12 @@ outputSections:
   description: Generic group for outputs
 outputs: []
 tests:
+- Test Playbook - Impossible Traveler - Enrichment
```

Comment:
Please remove the following test playbooks:
- Test Playbook - Impossible Traveler - Enrichment
- T1059 - Command and Scripting Interpreter - Test
- Test Playbook - IOC Alert